### DISCUSSION TITLE
在翻译窗口，是否可以支持各个服务的多列布局

### DISCUSSION THREAD

**Discussion by ZHAODayou** [pot-app/pot-desktop#741] [Ideas]
在翻译窗口中，各项服务的翻译结果目前是纵向的串行依次排列。
在个人的使用中，一般翻译结果每行较短，但行数较多。所以希望可以有办法把多个服务的小窗口变成并列的多列布局。

谢谢！

**Comment by imac8t** [comment]
确实
